charts/prometheus-operator-admission-webhook/templates/hooks/job-createSecret.yaml

```diff
@@ -10,15 +10,12 @@ metadata:
   name:  {{ template "prometheus-operator-admission-webhook.fullname" . }}-create
   namespace: {{ template "prometheus-operator-admission-webhook.namespace" . }}
   annotations:
-    "helm.sh/hook": pre-install,pre-upgrade
-    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
-    {{- with .Values.jobs.annotations }}
-    {{- toYaml . | nindent 4 }}
-    {{- end }}
+    {{- $defaultAnnotations := dict "helm.sh/hook" "pre-install,pre-upgrade" "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded" }}
+    {{- mustMerge .Values.jobs.createSecret.annotations .Values.jobs.annotations $defaultAnnotations | toYaml | nindent 4 }}
```

Comment:
@zeritti what do you think of this approach?  Before I finish fleshing it out

Reply:
That looks good (apart from the colon in the second key). I would suggest using named (helper) templates where reasonable to avoid repeating the same code in the files.

Reply:
Okay fixed everything.  Just haven't added descriptions to the comments I added.  Really struggling to describe it in any way that's even remotely brief.